content/docs/howto/php.md

```diff
@@ -27,147 +27,410 @@ for how to run the app.
 system libraries only present in the Full builder.**
 
-## Install a Specific PHP Version
+## Configure PHP
 
-The PHP Dist CNB allows you to specify a version of PHP to use during
+### Install a Specific PHP Version
+
+The [PHP Dist CNB][bp/php-dist] allows you to specify a version of PHP to use during
 deployment. This version can be specified in a number of ways, including
-through `buildpack.yml` or `composer.json` files. When specifying a
+through the `BP_PHP_VERSION` environment variable, `composer.lock`, or `composer.json` file. When specifying a
 version of PHP, you must choose a version that is available
-within the buildpack.
+within the buildpack. The supported versions can be found on the Paketo PHP
+Dist [releases page][release/php-dist].
 
 The buildpack prioritizes the versions specified in
 each possible configuration location with the following precedence, from
-highest to lowest: `buildpack.yml`, `composer.json`.
+highest to lowest: `BP_PHP_VERSION`, `composer.json`, `composer.lock`.
 
-### Using buildpack.yml
+#### Set the version via `BP_PHP_VERSION`
 
-To configure the buildpack to use PHP version when deploying your app,
-include the values like below in your `buildpack.yml` file. Any valid semver
-constraints are acceptable.
+To configure the buildpack to use a specific PHP version when deploying your app, set the
+following environment variable at build time, either directly (ex. `pack build
+my-app --env BP_PHP_VERSION=8.0.0`) or through a
+[project.toml](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md)
+file:
 
 {{< code/copyable >}}
----
-php:
-  version: 7.2.*
+BP_PHP_VERSION="8.0.0"
 {{< /code/copyable >}}
 
-### Using composer.json
+#### Set the version via a `composer.json` file
 
 If your apps use `composer`, you can specify the PHP version your apps use
 during deployment by configuring the `require` field in the `composer.json`
-file. To configure the buildpack to use PHP v7.1 or greater when deploying your
+file. To configure the buildpack to use PHP v8.0 or greater when deploying your
 app, include the values below in your `composer.json` file:
 
 {{< code/copyable >}}
 {
   "require": {
-    "php": ">=7.1"
+    "php": ">=8.0"
   }
 }
 {{< /code/copyable >}}
 
 If your app has a `composer.lock` file, the buildpack will use
 the php version defined there.
 
+#### Unsupported: Using buildpack.yml
+
+Specifying the PHP version through `buildpack.yml`  has been
+removed as of PHP language family buildpack v1.0.0.  To migrate from using
+`buildpack.yml` please set the `$BP_PHP_VERSION` environment variable.
+
+### Configure the PHP library directory
+
+In order to modify the default PHP include path in the PHP `.ini`
+configuration (where PHP will look for files to include), users can append
+onto the default libray code directory of `lib` for PHP by setting the
+`BP_PHP_LIB_DIR` environment variable at build-time.
```

Comment:
```suggestion
To add directories to the [`include_path`](https://www.php.net/manual/en/ini.core.php#ini.include-path) set in the default `php.ini`, buildpack users can set the
`BP_PHP_LIB_DIR` environment variable at build-time.
```

Is this an accurate paraphrase of what the buildpack's doing? I figured it'd be easier to link out to PHP docs 🙂

Reply:
yes!